app/build.gradle

```diff
@@ -34,4 +34,6 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+
+    implementation 'net.objecthunter:exp4j:0.4.8'
```

Comment:
Thư viện này để làm gì vậy bạn ?

Reply:
Thư viện này hỗ trợ em tính toán biểu thức ạ. Người ta cũng sử dụng stack để hỗ trợ tính toán. Nếu tự code thì em cũng sẽ dùng stack để tính toán như từng làm với các bài tính toán biểu thức trung hố hậu tố trong C++.